packages/test/test-end-to-end-tests/.vscode/launch.json

```diff
@@ -146,7 +148,8 @@
 				"fluid__test__driver": "r11s",
 				"fluid__test__backCompat": "${input:backCompat}",
 				"fluid__test__r11sEndpointName": "${input:r11sEndpoint}",
-				"FLUID_TEST_VERBOSE": "${input:verboseConsoleOutputt}",
+				"FLUID_TEST_VERBOSE": "${input:verboseConsoleOutput}",
```

Comment:
@Abe27342 I guess this will block the fallthrough to the local environment variables, so probably shouldn't merge this as-is.  Any ideas on how to organize/implement this to make it quick/easy to plug them in on CodeSpaces or another place where getkeys hasn't been run on the local machine?